### ISSUE TITLE
Be able to add non-patient as related person

### ISSUE THREAD

**Issue by jackcmeyer** [HospitalRun/hospitalrun-frontend#1947]
## 🚀 Feature Proposal

Currently, HospitalRun v2 requires a related person to be a patient. However, a related person may not be a patient. HospitalRun should support non-patients being a related person.

## Motivation

Better tracking of patient data.

**Comment by jackcmeyer** [comment]
Opening this issue to start a conversation in hopes of gaining ideas and feedback.

**Comment by bmoore235** [comment]
Does directly linking patient records constitute a health privacy law violation? Would this goal be better accomplished by isolated family history information on a per patient basis?

**Comment by kumikokashii** [comment]
What sorts of information would a related person add to the patient? I wonder if this is more like an emergency contact information, or rather related to the health conditions of the patient?

**Comment by kevinhaube** [comment]
A parallel experience to this is when I used to work at a gym and we'd register what we called "prospect" accounts as guest accounts for members who had access for a +1. There was a limited amount of information needed: name, birthday, phone number, relationship, and a barcode to look them up by (which usually was just their 10 digit phone number). I was just looking at the process of adding a Related Person. It could easily show up as a static option at the bottom of the autocomplete list, something like "Add a Non-Patient", or a simple checkbox labeled "Related Person is not a patient", and upon selecting it, a secondary body could show up on the Modal that has the appropriate fields.

If we know the fields needed, and it's deemed necessary, I wouldn't mind giving it a shot!
